updates/version.yaml

```diff
@@ -49,3 +49,6 @@
 1.3.7: Fix config reference in LocaleMiddleware
 1.3.8: Keep query string when switching locales
 1.4.0: Add importer and exporter for messages
+1.4.1:
```

Comment:
Is this form of release notes actually supported by October? Typically multi-line versions are to provide one comment about the version and any DB migration scripts to run.

Reply:
You may be right, I'll change this.